src/core/logical_layers/utils/logicalLayerFabric.ts

```diff
@@ -218,6 +223,17 @@ export function createLogicalLayerAtom(
         }
       });
 
+      onAction('edit', async () => {
+        try {
+          if (!state?.isEditable) return;
+          if (state.source?.style?.type === 'mcda') {
+          }
```

Comment:
do we need empty if statement?

Reply:
It's not needed! And also the action handler it's in not needed too. I'm gonna clean this up, thanks!

Reply:
Removed along with unused action